src/Message/AbstractRequest.php

```diff
@@ -695,6 +715,40 @@ public function getDebtorId()
     {
         return $this->getParameter('debtorId');
     }
+    
+    public function getCustomerIsPresent()
+    {
+        return $this->getParameter('customer_is_present');
+    }
+
+    /**
+     * is Customer present by payment
+     */
+    public function setCustomerIsPresent($isPresent)
+    {
+        if (!is_bool($isPresent)) {
```

Comment:
Personally I would have left it as boolean until it was needed in `getCustomerIsPresent()` (or later - where the message is actually constructed). The yes/no is very much about formatting the message, and so best left until the last point, while `customerIsPresent` is a boolean business requirement.

But this will work :-)